apps/web/app/(app)/gardens/[chain]/[garden]/create-community/page.tsx

```diff
@@ -1,42 +1,50 @@
+"use client";
 import {
   TokenGarden,
   getCommunityCreationDataDocument,
   getCommunityCreationDataQuery,
 } from "#/subgraph/.graphclient";
 import { CommunityForm } from "@/components/Forms";
-import { initUrqlClient, queryByChain } from "@/providers/urql";
-import React from "react";
+import LoadingSpinner from "@/components/LoadingSpinner";
+import useSubgraphQueryByChain from "@/hooks/useSubgraphQueryByChain";
+import React, { useEffect } from "react";
 import { Address } from "viem";
 
-export default async function page({
+export default function Page({
   params: { chain, garden },
 }: {
   params: { chain: number; garden: string };
 }) {
-  const { urqlClient } = initUrqlClient();
-
-  const { data: result, error: error } =
-    await queryByChain<getCommunityCreationDataQuery>(
-      urqlClient,
+  const { data: result, error: getCommunityCreationDataQueryError } =
+    useSubgraphQueryByChain<getCommunityCreationDataQuery>(
       chain,
       getCommunityCreationDataDocument,
       { addr: garden },
     );
 
+  useEffect(() => {
+    if (getCommunityCreationDataQueryError) {
+      console.error(
+        "Error while fetching community creation data: ",
+        getCommunityCreationDataQueryError,
+      );
+    }
+  }, [getCommunityCreationDataQueryError]);
+
   const registryFactoryAddr = result?.registryFactories?.[0].id as Address;
   const tokenGarden = result?.tokenGarden as TokenGarden;
   const alloContractAddr = result?.tokenGarden?.communities?.[0]
     .alloAddress as Address;
 
-  return (
+  return tokenGarden ? (
     <div className="mx-auto flex max-w-[820px] flex-col items-center justify-center gap-4">
       <div className="text-center sm:mt-5">
         <h2 className="text-xl font-semibold leading-6 text-gray-900">
           Welcome to the {tokenGarden?.symbol} Community Form!
         </h2>
         <div className="mt-1">
           <p className="text-sm">
-            Create a vibrant community around the {tokenGarden.name} by
+            Create a vibrant community around the {tokenGarden?.name} by
             providing the necessary details below.
```

Comment:
should been removing '?' instead adding, already you check it in line 39? mark as solved its not the case.

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/36d5d5c6e42e074f66dfb1030edfe1d7d80665db